ui/src/Components/ProfilePage.jsx

```diff
@@ -112,39 +133,83 @@ export default class ProfilePage extends React.Component {
           {!this.state.profile && this.state.loading == false ? (
             <div> Profile Does Not Exist </div>
           ) : (
-            <div className="row-buffer side-buffer left-align">
-              <h4>{this.state.profile.result.first_name} {' '}
-              {this.state.profile.result.last_name}</h4>
-              {this.state.profile.result.city ? <H4>Servicing the {this.state.profile.result.city} area</H4> : <div/>}
-              <Table className="row-buffer" responsive>
+            <div className="row-buffer side-buffer">
+               {this.state.profile.result.app_status == 'Began Application' ? (
+                  <Alert className="width-fit" color="primary">
+                    Your application is in draft mode.
+                    Submit your application to employers through Edit Profile > Submit
+                  </Alert>                    
+                  ) : <div/>}
+                <Alert className="push-right" color="success">
+                  APPLICATION STATUS: {this.state.profile.result.app_status}
+                </Alert>
+                <h4 className="push-left">{this.state.profile.result.first_name} {' '}
+                {this.state.profile.result.middle_name} {' '}
+                {this.state.profile.result.last_name}</h4>
+              <div>
+              </div>
+              <Table responsive>
                 <tbody>
                 <tr>
-                    <td>Email</td>
+                    <th>Email</th>
                     <td>{this.state.profile.result.email}</td>
                   </tr>
                   <tr>
-                    <td>Phone Number</td>
-                    <td>{this.state.profile.result.contact_num}</td>
+                    <th>Phone Number</th>
+                    {this.state.profile.result.contact_ext ? (
+                    <td>{this.state.profile.result.contact_num.substring(0,3)}-{this.state.profile.result.contact_num.substring(3,6)}-{this.state.profile.result.contact_num.substring(6,10)}  x  {this.state.profile.result.contact_ext}</td>
+                  ) : <td>{this.state.profile.result.contact_num}</td>}
+                  </tr>
+                  <tr>
+                    <th>Address</th>
+                    {this.state.profile.result.address ? (
+                        <td><div>{this.state.profile.result.address}</div>
+                        <div>{this.state.profile.result.city},  {this.state.profile.result.postal_code.substring(0,3)} {this.state.profile.result.postal_code.substring(3,6)}   {this.state.profile.result.province}</div>
+                        </td>
+                      ) : <td>Not Provided</td>}
+                  </tr>
+                  <tr>
+                    <th>Date of Birth (MM/DD/YYYY)</th>
+                    {this.state.profile.result.dob ? (
+                      <td>{this.state.profile.result.dob.substring(0,2)}/{this.state.profile.result.dob.substring(2,4)}/{this.state.profile.result.dob.substring(4,8)}</td>
```

Comment:
Just a thought but given the number of times we call `this.state.profile.result` we should probs extract it out to a variable. We're only fetching and setting this.state.profile once, so I think that should be fine.

Reply:
good point, maybe we'll want to eventually fix this redundancy across all the JSX files, in one sweep

Reply:
that works with me